react-validated-input-component/dist/styles.css

```diff
@@ -0,0 +1,34 @@
+.container {
+  display: flex;
+  flex-direction: column;
+}
+
+.input-box {
+  display: flex;
+  flex-direction: row;
+}
+
+.fa-check {
+  color: greenyellow;
+}
+
+.fa-times {
+  color: red;
+}
```

Comment:
It is bad practice to directly change the behavior of a class from a CSS library. This can cause confusion with other people working on the project or even yourself later on, if the class doesn't behave as expected. A better approach is to use classes you define to make the desired changes in conjunction with the classes from the CSS library. For example her you could do:

```CSS
.text-green {
  color: greenyellow;
}

.text-red {
  color: red;
}
```